__tests__/index.test.js

```diff
@@ -75,6 +75,25 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
+
+  test("GET 200: returns article with the article ID passed with an added key of comment_count", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article[0]).toMatchObject({
```

Comment:
The endpoint should just send back the single article object rather than an array containing one article